docs/src/main/asciidoc/redis-reference.adoc

```diff
@@ -278,6 +278,10 @@ As mentioned above, the API is divided into groups:
 - string - `.value(valueType)`
 - transactions - `withTransaction`
 - json - `.json()` (requires the https://redis.com/modules/redis-json/[redis-json] module on the server side)
+- bloom - `.bloom()` (requires the https://redis.com/modules/redis-bloom/[redis-bloom] module on the server side)
+- cuckoo - `.cuckoo()` (requires the https://redis.com/modules/redis-bloom/[redis-bloom] module on the server side, which also provides the cuckoo filter commands)
+- count-min - `.countmin()` (requires the https://redis.com/modules/redis-bloom/[redis-bloom] module on the server side, which also provides the count-min filter commands)
+- top-k - `.topk()` (requires the https://redis.com/modules/redis-bloom/[redis-bloom] module on the server side, which also provides the top-k filter commands)
```

Comment:
It'll be very useful to show example usages of these but probably something for the followup PR

Reply:
I would rather redirect to the Redis documentation. 
Before using these commands (and associated structures), you must have some ideas and mathematical knowledge. Quarkus documentation should not become a math lesson. 

There are some basic usage examples in the test. I can copy snippets, but again, that won't be very useful if you don't understand the structure. (add items, execute some operations and receives opaque results)

Reply:
Yes, the comment was more about adding the Quarkus related code snippets of course with the assumption that ones understand the underlying redis workings

Reply:
I will write something once I'm done with all the redis labs groups (... never ending task :-D)

Reply:
Sounds good. Thanks a lot @cescoffier for the improvements!